backend/private-graph/graph/schema.resolvers.go

```diff
@@ -46,14 +46,14 @@ import (
 	"github.com/highlight-run/highlight/backend/util"
 	"github.com/highlight-run/highlight/backend/vercel"
 	"github.com/highlight-run/highlight/backend/zapier"
-	"github.com/highlight/highlight/sdk/highlight-go"
+	highlight "github.com/highlight/highlight/sdk/highlight-go"
```

Comment:
#5183 should have failed `make check` (that runs `make private-gen`) but for some reason it was skipped. Not sure why but that's why this import change is here.